Pause Teleporter address interaction (#185)
ava-labs/teleporter: contracts/src/Teleporter/upgrades/README.md

    }
```

> For sending messages with the Teleporter registry, dapps should generally use `TeleporterUpgradeable._getTeleporterMessenger`, which by default will use the latest version. If the dapp wants to send a message through a specific Teleporter version, it can override `_getTeleporterMessenger()` to use the specific Teleporter version with  `TeleporterRegistry.getTeleporterFromVersion`.

Reviewer: Can you add here how TeleporterUpgradeable._getTeleporterMessenger extends TeleporterRegistry.getLatestTeleporter?

Using latest version:

```solidity
        ITeleporterMessenger teleporterMessenger = _getTeleporterMessenger();
```

Using specific version:

```solidity
        // Override _getTeleporterMessenger to use specific version.
        function _getTeleporterMessenger() internal view override returns (ITeleporterMessenger) {
            ITeleporterMessenger teleporter = teleporterRegistry
                .getTeleporterFromVersion($VERSION);
            require(
                !pausedTeleporterAddresses[address(teleporter)],
                "TeleporterUpgradeable: Teleporter sending version paused"
            );

            return teleporter;
        }

        ITeleporterMessenger teleporterMessenger = _getTeleporterMessenger();
```

## Pausing Teleporter interactions

> Dapps that inherit from `TeleporterUpgradeable` can pause Teleporter interactions by calling `TeleporterUpgradeable.pauseTeleporterAddress`. This pause function stops receiving Teleporter message from the paused Teleporter address, and also makes sure to not send messages through the paused Teleporter address in `_getTeleporterMessenger()`. 

Reviewer: ```suggestion
Dapps that inherit from `TeleporterUpgradeable` can pause Teleporter interactions by calling `TeleporterUpgradeable.pauseTeleporterAddress`. This function prevents the contract from interacting with the paused Teleporter  address via `_getTeleporterMessenger()`. 
```

This section is worded a bit awkwardly. I suggested a change to clarify things.

Reviewer 2: The suggestion seems more clear, but there's an extra space between `Teleporter  address`
